StoreApp.BLL/Services/OrderDetailService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    class OrderDetailService : IOrderDetailService
+    {
+        IUnitOfWork Database;
+        public OrderDetailService(IUnitOfWork eof)
+        {
+            Database = eof;
+        }
```

Comment:
The methods must be splitted by using indents

Reply:
fixed